app/build.gradle

```diff
@@ -31,7 +31,7 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
         debug {
-            minifyEnabled true
+            minifyEnabled false
```

Comment:
why we change it from true to false? @javiersuweijie

Reply:
cant debug properly if we minify it during debug